cmd/svcat/output/instance.go

```diff
@@ -54,8 +54,8 @@ func writeInstanceListTable(w io.Writer, instanceList *v1beta1.ServiceInstanceLi
 		t.Append([]string{
 			instance.Name,
 			instance.Namespace,
-			instance.Spec.GetSpecifiedClass(),
-			instance.Spec.GetSpecifiedPlan(),
+			instance.Spec.GetSpecifiedClusterServiceClass(),
```

Comment:
I'm not sure that this part is right. The code here is printing out the associated class and plan with an instance. So whether it is a namespaced class or a cluster scoped class, the name of the class should be printed regardless.

I think we should make a new function (probably just in this pkg and not in the api types) called `GetClassName` or something,  and it would handle figuring out which type of class it is, and returning its name.